app/main.py

```diff
@@ -1 +1,54 @@
 #  write your code here
+class Animal:
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
```

Comment:
Place each argument on a new line including `self`, so that there would be no arguments on the same lines with opening and closing parenthesis.